openedx/adg/admin_dashboard/applications/helpers.py

```diff
@@ -224,3 +233,52 @@ def get_application_review_url(application_id, query_params_string):
         f'{application_review_url}?{query_params_string}' if query_params_string else application_review_url
     )
     return application_review_url
+
+
+def get_valid_date(day, month, year):
+    """
+    Convert day/month/year to a date object.
+
+    Arguments:
+        day (string): day of the date
+        month (string): month of the date
+        year (string): year of the date
+
+    Returns:
+        _date: date object if it is valid, otherwise None
```

Comment:
_date is not a type, let's call it `datetime.date` or `date`

Reply:
@javeriaejaz-arbisoft this still needs to be updated according to [google format docstring](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html).